openpgp/read.go

```diff
@@ -46,6 +47,7 @@ type MessageDetails struct {
 	DecryptedWith            Key                 // the private key used to decrypt the message, if any.
 	IsSigned                 bool                // true if the message is signed.
 	SignedByKeyId            uint64              // the key id of the signer, if any.
+	SignedByFingerprint		 []byte				 // the key fingerprint of the signer, if any. (only v6)
```

Comment:
If you want you could get the signer's fingerprint using the Issuer Fingerprint subpacket, if present, but that's orthogonal from whether the PKESK is v6.